docker/ltcd/start-ltcctl.sh

```diff
@@ -43,8 +43,12 @@ RPCUSER=$(set_default "$RPCUSER" "devuser")
 RPCPASS=$(set_default "$RPCPASS" "devpass")
 NETWORK=$(set_default "$NETWORK" "simnet")
 
-exec ltcctl \
-    "--$NETWORK" \
+PARAMS=""
+if [ "$NETWORK" != "mainnet" ]; then
+    PARAMS=$(echo --$NETWORK)
```

Comment:
No need to use `echo`. This line can just be:
`PARAMS="--$NETWORK"`

Reply:
OK! I will also change that for `btcd/start-btcctl.sh`.